### ISSUE TITLE
[Bug]: Unable to print a map with WMS layer from Third Party server

### ISSUE THREAD

**Issue by Mavialle** [3liz/lizmap-web-client#3533]
### What is the bug?

When a project contains a WMS layer and Third Party server option is activated in Lizmap plugin for this layer, print is impossible in Lizmap.

### Steps to reproduce the issue

1. Create a new QGIS project (.qgs file)
2. Open layer Plan IGN v2 from https://wxs.ign.fr/essentiels/geoportail/r/wms
3. Zoom somewhere, approximately until 1:200000
4. Open Lizmap plugin and set map extent from map canvas + active print option. Then, go to layer menu, select Plan IGN v2 layer and active select Third Party WMS layers Option + toggled option .
5. Create a new layout just with map object
6. Save projet and put it on Lizmap server
7. Go to the new map on Lizmap and start a print --> nothing happens, print preparation takes forever

If Third Party WMS layers Option is removed in QGIS with Lizmap plugin, print is OK on Lizmap.

Maybe Upgrade Lizmap version can be solve the issue, but I see nothing in release description about something who looks like to this issue..

### Versions

**Versions :**

* Lizmap Web Client : 3.5.7
* Lizmap plugin : 3.9.8
* QGIS Desktop : 3.22.14
* QGIS Server : 3.22.14
* Py-QGIS-Server : not used
* QGIS Server plugin atlasprint : 3.3.1
* QGIS Server plugin cadastre : 1.17.1
* QGIS Server plugin lizmap_server : 1.2.2
* QGIS Server plugin wfsOutputExtension : 1.7.0

### Check Lizmap plugin

- [X] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### Operating system

Ubuntu

### Browsers

Chrome

### Browsers version

Version 110.0.5481.178 (Build officiel) (64 bits)

**Comment by Mavialle** [comment]
I've found this documentation : https://docs.qgis.org/3.22/en/docs/server_manual/services/wms.html#wms-external

Maybe it's a QGIS server issue, and maybe I just need to add  EXTERNAL_WMS parameter, but I don't understand how ?

**Comment by Mavialle** [comment]
Ok I got it.

QGIS Server seems to not find the layer called by Third Party.  The browser say that the layer XXXX (= name of layer in wms service) is not found.

So, I duplicated the wms layer : one called  by Third Party, the other not but named XXXX and not displayed (I put it in a group of layer named "hidden"). And when the GetPrint request is sent, QGIS server use the layer not called by third party and it's work.

Have a good day !
